frontend/src/features/admin-form/create/logic/components/LogicContent/EditLogicBlock/EditCondition/EditConditionBlock.tsx

```diff
@@ -180,6 +180,16 @@ export const EditConditionBlock = ({
     }
   }, [currentSelectedField])
 
+  const logicValueWrapperWidth = useMemo(() => {
+    if (!ifValueTypeValue) return 'auto'
```

Comment:
Is this line needed because the typescript linter is complaining? If not, the `switch`'s `default` block below already captures the falsy case, so the line here is redundant.

In fact, since the change is for a specific check, I suppose the whole memo could be simplified to:

```
return ifValueTypeValue === LogicIfValue.MultiSelect ? '0px' : 'auto'
```

Reply:
Yep, you're right!